src/frontend/query_commands.ml

```diff
@@ -33,6 +33,10 @@ module Printtyp = Type_utils.Printtyp
 
 exception No_nodes
 
+let () = Location.register_error_of_exn (function
+  | No_nodes -> Some (Location.error "No nodes")
```

Comment:
I should have noticed this in another PR, but perhaps instead of `No_nodes` we could reuse `Destruct.Nothing_to_do`?